common/src/main/java/org/opensearch/ml/common/transport/register/MLRegisterModelInput.java

```diff
@@ -372,6 +396,7 @@ public static MLRegisterModelInput parse(XContentParser parser, String modelName
         Boolean doesVersionCreateModelGroup = null;
         Boolean isHidden = null;
         Guardrails guardrails = null;
+        Map<String, String> modelInterface = new HashMap<>();
```

Comment:
Init it to null to avoid meaningless object creation since `filterInterfaceMap` will always return an object.

Reply:
Changed